resin/models/environment_variables.py

```diff
@@ -228,11 +228,11 @@ def create(self, app_id, env_var_name, value):
             value (str): environment variable value.
 
         Returns:
-            str: new application environment info.
+            dict: new application environment info.
```

Comment:
Small nit, but should this say "application environment variable info" instead of "application environment info".